integrated_channels/blackboard/exporters/content_metadata.py

```diff
@@ -22,7 +22,27 @@ class BlackboardContentMetadataExporter(ContentMetadataExporter):
         'course_child_content_metadata': 'course_child_content_metadata',
     }
 
-    DESCRIPTION_TEXT_TEMPLATE = "<a href={enrollment_url}>Go to edX course page</a><br />"
+    DESCRIPTION_TEXT_TEMPLATE = "<a href={enrollment_url}>Go to edX course page</a><br/>"
```

Comment:
should these names match what are in the html template string more or less (e.g. go_to_link?)